### PR TITLE
Correcting the order of opreation. 

### PR DESCRIPTION
**Pull request by JerryNixon**
Not really used in UWP, but still, now it is correct.

### CODE REVIEW

File: Source/Windows10/Prism.Windows/Navigation/INavigationParametersExtensions.cs [hunk 1]
```diff
@@ -29,14 +29,14 @@ public static NavigationMode GetNavigationMode(this INavigationParameters parame
             return (parameters as INavigationParametersInternal).GetValue<NavigationMode>(nameof(NavigationMode));
         }
 
-        public static INavigationService GetNavigationService(this INavigationParameters parameters)
+        public static IPlatformNavigationService GetNavigationService(this INavigationParameters parameters)
```
Comment (brianlagunas) [inline Source/Windows10/Prism.Windows/Navigation/INavigationParametersExtensions.cs]: YOu should leave this as is. No need to return the full service. We expose platform methods via extension methods.

Comment (JerryNixon) [reply Source/Windows10/Prism.Windows/Navigation/INavigationParametersExtensions.cs]: Re-casting is the first thing we have to do every single time. This method is used in one place, only: ViewModel.OnNavigatedTo where we use methods like GoForward() and Refresh() which are only visible in the Platform version of this interface. The developer doesn't have a practical need for this interface.

Comment (brianlagunas) [reply Source/Windows10/Prism.Windows/Navigation/INavigationParametersExtensions.cs]: Developers can still call GoForward and Refresh off of the INavigationService interface

**Comment by JerryNixon** [comment]
K, all done.

**Pull request merged**